ldm/generate.py

```diff
@@ -348,9 +351,30 @@ def process_image(image,seed):
         width = width or self.width
         height = height or self.height
         
+        def _conv_forward_asymmetric(self, input, weight, bias):
```

Comment:
`generate.py` has gotten very large, and I wonder if it makes sense to move this and the rest of the seamless tiling code into a module named something like `ldm/invoke/seamless.py` ?

Reply:
I moved it there in 99581db, let me know if you want it structured or named differently.